src/vue/vuex-modules/data/auth-module.js

```diff
@@ -31,22 +35,20 @@ AuthModule.store = (moduleInstance) => {
         count: 0,
         text: null
       },
-      showUI: moduleInstance._showUIDefault,
-      enableLogin: moduleInstance._showUIDefault, // this doesn't change based upon auth
-      promptLogin: !!moduleInstance._auth // don't prompt for login if we have no auth object
+      externalLoginUrl: moduleInstance._externalLoginUrl,
+      externalLogoutUrl: moduleInstance._externalLogoutUrl,
+      enableLogin: !!moduleInstance._auth // don't enable login if we have no auth object
```

Comment:
I'm thinking that maybe now it's better to use `Boolean(moduleInstance._auth)` instead of `!!moduleInstance._auth`. I think Boolean is easier to read and is less hacky. What do you think?

Reply:
yes, that's a good idea. will fix.